services/api/db/admin.py

```diff
@@ -0,0 +1,30 @@
+from django.contrib import admin
+from db.models import User
+from django.contrib.auth.admin import UserAdmin as BaseUserAdmin
+
+class UserModelAdmin(BaseUserAdmin):
```

Comment:
Admin panel is not a part of the plan. Pls sort it out and remove it , if not applicable